content/chat/rooms/messages.textile

```diff
@@ -319,12 +370,18 @@ The updated message response is identical to the structure of a message, with th
 
 h2(#delete). Delete a message
 
-<aside data-type='new'>
-<p>Message deletion was added in version 0.3. It is only available in the JavaScript and React SDKs.</p>
-</aside>
+blang[javascript,react].
+  <aside data-type='new'>
+  <p>Message deletion was added in version 0.3.</p>
+  </aside>
 
-blang[javascript].
-  Use the "@messages.delete()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#delete method to delete a message in a chat room. All users that are "subscribed":#subscribe to messages on that room will receive the deletion:
+blang[swift,kotlin].
+  <aside data-type='new'>
+  <p>Message deletion was added in version 0.2.0.</p>
+  </aside>
+
+blang[javascript,swift,kotlin].
+  Use the <span lang="javascript">"@messages.delete()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#delete</span><span lang="swift">"@messages.delete()@":https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/messages/delete(message:params:)</span><span lang="kotlin">"@messages.delete()@":https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat-android/com.ably.chat/-messages/delete.html</span> method to delete a message in a chat room. All users that are "subscribed":#subscribe to messages on that room will receive the deletion:
```

Comment:
In Swift, there's an extra bracket on the end of here

Reply:
Fixed